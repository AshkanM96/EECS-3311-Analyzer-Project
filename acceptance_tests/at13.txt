difference
open_set_enumeration
open_set_enumeration
integer_constant(1)
close_set_enumeration
integer_constant(6)
close_set_enumeration
open_set_enumeration
open_set_enumeration
integer_constant(1)
close_set_enumeration
close_set_enumeration
evaluate
