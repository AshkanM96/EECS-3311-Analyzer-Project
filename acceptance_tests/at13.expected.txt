  Expression currently specified: ?
  Report: Expression is initialized.
->difference
  Expression currently specified: (? \ nil)
  Report: OK.
->open_set_enumeration
  Expression currently specified: ({?} \ nil)
  Report: OK.
->open_set_enumeration
  Expression currently specified: ({{?}} \ nil)
  Report: OK.
->integer_constant(1)
  Expression currently specified: ({{1, ?}} \ nil)
  Report: OK.
->close_set_enumeration
  Expression currently specified: ({{1}, ?} \ nil)
  Report: OK.
->integer_constant(6)
  Expression currently specified: ({{1}, 6, ?} \ nil)
  Report: OK.
->close_set_enumeration
  Expression currently specified: ({{1}, 6} \ ?)
  Report: OK.
->open_set_enumeration
  Expression currently specified: ({{1}, 6} \ {?})
  Report: OK.
->open_set_enumeration
  Expression currently specified: ({{1}, 6} \ {{?}})
  Report: OK.
->integer_constant(1)
  Expression currently specified: ({{1}, 6} \ {{1, ?}})
  Report: OK.
->close_set_enumeration
  Expression currently specified: ({{1}, 6} \ {{1}, ?})
  Report: OK.
->close_set_enumeration
  Expression currently specified: ({{1}, 6} \ {{1}})
  Report: OK.
->evaluate
  Expression currently specified: ({{1}, 6} \ {{1}})
  Report: Error (Expression is not type-correct).
